### DISCUSSION TITLE
update the version of axmol engine

### DISCUSSION THREAD

**Discussion by PedroQuesado** [axmolengine/axmol#2228] [General]
to update the axmol to last version i only have to fetch from repository and pull the all comits to my local repository ?

**Comment by rh101** [comment]
If you have a fork on GitHub, you can just use the sync functionality available in the UI. It's documented [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork).

You can update your local repo by using something like the following: 
```
git fetch --all
git pull
```

**Reply by PedroQuesado** [reply]
ok 
what i have to do to update my game with new version of axmol engine
i have to run the setup again

**Reply by rh101** [reply]
That depends on a few factors.  

If it's a minor update, then you should be able to just recompile your application.

If it's an update with lots of changes, then check if the template projects have changed, and merge in whatever is required, and yes, run setup again, just in case.

**Reply by PedroQuesado** [reply]
i upgrade axmol engine  from 2.2.0 to 2.2.1

**Reply by rh101** [reply]
That is a minor version update, so you shouldn't need to do anything besides re-compiling your project.

**Comment by aismann** [comment]
My update strategie is often:
Delete the current axmol fork (with all old PRs etc) and start a new clean fork.

**Comment by aismann** [comment]
@PedroQuesado 
Closing?
